src/tests/unit/tests/electron/views/screenshot/__snapshots__/screenshot-view.test.tsx.snap

```diff
@@ -7,27 +7,49 @@ exports[`ScreenshotView renders when passed a value for screenshotData 1`] = `
   >
     Target Page Screenshot
   </h1>
-  <Screenshot
-    altText="axe-android results screenshot with highlighted components"
-    encodedImage="testScreenshotData"
+  <ScreenshotContainer
+    screenshotData={
+      Object {
+        "base64PngData": "test-base-64-png-data",
+      }
+    }
   />
 </div>
 `;
 
 exports[`ScreenshotView renders when passed empty screenshotData {} 1`] = `
 <div>
-  Screenshot for scan is unavailable
+  <h1
+    tabIndex={0}
+  >
+    Target Page Screenshot
+  </h1>
+  <ScreenshotContainer
+    screenshotData={Object {}}
```

Comment:
I think these empty cases in `screenshot-view.test.tsx` are probably obsolete and can be removed now that the "screenshot is unavailable" message responsibility has moved to the screenshot container

Reply:
Removed obsolete "empty screenshot" test cases